src/test/java/com/wikia/webdriver/testcases/auth/SignupTests.java

```diff
@@ -229,12 +230,17 @@ private RegisterPage navigateToSignUpOnMobile(TopBar topBar) {
       .navigateToSignUp();
   }
 
-  private RegisterPage openSignUpModalOnDesktop() {
-    return new DetachedRegisterPage(new NavigationBar().clickOnRegister());
+//  private RegisterPage openSignUpModalFromGlobalavOnDesktop() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)